sdk/mobile-center/src/main/java/com/microsoft/azure/mobile/MobileCenter.java

```diff
@@ -281,42 +281,42 @@ private synchronized boolean instanceConfigure(Application application, String a
         if (mApplication != null) {
             MobileCenterLog.warn(LOG_TAG, "Mobile Center may only be configured once");
             return false;
-        }
-        if (application == null) {
+        } else if (application == null) {
             MobileCenterLog.error(LOG_TAG, "application may not be null");
-            return false;
-        }
-        if (appSecret == null || appSecret.isEmpty()) {
+        } else if (appSecret == null || appSecret.isEmpty()) {
             MobileCenterLog.error(LOG_TAG, "appSecret may not be null or empty");
-            return false;
+        } else {
+            mApplication = application;
+
+            /* If parameters are valid, init context related resources. */
+            StorageHelper.initialize(application);
+            mServices = new HashSet<>();
+
+            /* Init channel. */
+            mLogSerializer = new DefaultLogSerializer();
+            mChannel = new DefaultChannel(application, appSecret, mLogSerializer);
+            mChannel.setEnabled(isInstanceEnabled());
+            if (mServerUrl != null)
+                mChannel.setServerUrl(mServerUrl);
+            MobileCenterLog.logAssert(LOG_TAG, "Mobile Center SDK configured successfully.");
+            return true;
         }
-        mApplication = application;
-
-        /* If parameters are valid, init context related resources. */
-        StorageHelper.initialize(application);
-        mServices = new HashSet<>();
-
-        /* Init channel. */
-        mLogSerializer = new DefaultLogSerializer();
-        mChannel = new DefaultChannel(application, appSecret, mLogSerializer);
-        mChannel.setEnabled(isInstanceEnabled());
-        if (mServerUrl != null)
-            mChannel.setServerUrl(mServerUrl);
-        return true;
+
+        MobileCenterLog.logAssert(LOG_TAG, "Mobile Center SDK configuration failed.");
+        return false;
     }
 
     @SafeVarargs
     private final synchronized void startServices(Class<? extends MobileCenterService>... services) {
         if (services == null) {
-            MobileCenterLog.logAssert(LOG_TAG, "Cannot start services, services array is null. Failed to start Mobile Center SDK");
+            MobileCenterLog.error(LOG_TAG, "Cannot start services, services array is null. Failed to start services.");
```

Comment:
I agree but not this line. This is the case if `services == null`. I'll change the below log with class names.